scripts/bert/finetune_classifier.py

```diff
@@ -89,12 +89,17 @@
     type=float,
     default=5e-5,
     help='Initial learning rate, default is 5e-5')
+parser.add_argument(
+    '--epsilon',
+    type=float,
+    default=1e-08,
```

Comment:
use 1e-6 so that existing scripts are not broken

Reply:
Already reset.